recipes/physx/4.x.x/conanfile.py

```diff
@@ -254,25 +213,25 @@ def _get_target_build_platform(self):
         }.get(str(self.settings.os))
 
     def package(self):
-        cmake = self._configure_cmake()
-        cmake.install()
+        cmake = CMake(self)
+        cmake.install(build_type=self._get_physx_build_type())
 
-        tools.save(os.path.join(self.package_folder, "licenses", "LICENSE"), self._get_license())
+        save(self, os.path.join(self.package_folder, "licenses", "LICENSE"), self._get_license())
 
         out_lib_dir = os.path.join(self.package_folder, "lib", self._get_physx_build_type())
-        self.copy(pattern="*.a", dst="lib", src=out_lib_dir, keep_path=False)
-        self.copy(pattern="*.so", dst="lib", src=out_lib_dir, keep_path=False)
-        self.copy(pattern="*.dylib*", dst="lib", src=out_lib_dir, keep_path=False)
-        self.copy(pattern="*.lib", dst="lib", src=out_lib_dir, keep_path=False)
-        self.copy(pattern="*.dll", dst="bin", src=out_lib_dir, keep_path=False)
 
-        tools.rmdir(out_lib_dir)
-        tools.rmdir(os.path.join(self.package_folder, "source"))
+        copy(self, pattern="*.a", dst="lib", src=out_lib_dir, keep_path=False)
+        copy(self, pattern="*.so", dst="lib", src=out_lib_dir, keep_path=False)
+        copy(self, pattern="*.dylib*", dst="lib", src=out_lib_dir, keep_path=False)
+        copy(self, pattern="*.lib", dst="lib", src=out_lib_dir, keep_path=False)
+        copy(self, pattern="*.dll", dst="bin", src=out_lib_dir, keep_path=False)
```

Comment:
Not too sure what you mean by you were right, something was off. I made some changes locally (and it works) and now I uploaded them. Hope they're good now 🤞 🙂

Reply:
dst is still off you need ... but for all of them :)

Might be easier to patch the CMake install 🤔 

```suggestion
        copy(self, pattern="*.a", dst=os.path.join(self.package_folder, "lib"), src=out_lib_dir, keep_path=False)
        copy(self, pattern="*.so", dst="lib", src=out_lib_dir, keep_path=False)
        copy(self, pattern="*.dylib*", dst="lib", src=out_lib_dir, keep_path=False)
        copy(self, pattern="*.lib", dst="lib", src=out_lib_dir, keep_path=False)
        copy(self, pattern="*.dll", dst="bin", src=out_lib_dir, keep_path=False)
```

Reply:
@prince-chrismc thanks for the suggestion 🙂 I updated the `copy` method (hopefully it works now 🤞) since patching the CMake install would be quite "intricate" 🙂